### PR TITLE
test/e2e/framework/util.go:use the term 'Control Plane'

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

ref https://github.com/kubernetes/kubernetes/issues/94901

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e/cloud/gcp/cluster_upgrade.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func masterUpgrade(f *framework.Framework, v string) error {
 	case "gce":
 		return masterUpgradeGCE(v, false)
 	case "gke":
-		return framework.MasterUpgradeGKE(f.Namespace.Name, v)
+		return framework.MasterControlPlaneUpgradeGKE(f.Namespace.Name, v)
```
Comment (oomichi) [inline test/e2e/cloud/gcp/cluster_upgrade.go:98]: Need to remove `Master` from the function name.

File: test/e2e/framework/nodes_util.go [hunk 2]
```diff
@@ -75,8 +75,8 @@ func AppendContainerCommandGroupIfNeeded(args []string) []string {
 	return args
 }
 
-// MasterUpgradeGKE upgrades master node to the specified version on GKE.
-func MasterUpgradeGKE(namespace string, v string) error {
+// MasterControlPlaneUpgradeGKE upgrades master node to the specified version on GKE.
+func MasterControlPlaneUpgradeGKE(namespace string, v string) error {
```
Comment (oomichi) [inline test/e2e/framework/nodes_util.go:79]: Need to remove `Master` from the function name.

**Review by oomichi — COMMENTED** [review]
/cc @oomichi

File: test/e2e/cloud/gcp/cluster_upgrade.go (hunk 1 above)
Comment (tanjunchen) [reply test/e2e/cloud/gcp/cluster_upgrade.go:98]: Yes, I will change it immediately. Thank you.

File: test/e2e/framework/nodes_util.go (hunk 2 above)
Comment (tanjunchen) [reply test/e2e/framework/nodes_util.go:79]: fine

File: test/e2e/cloud/gcp/cluster_upgrade.go (hunk 1 above)
Comment (tanjunchen) [reply test/e2e/cloud/gcp/cluster_upgrade.go]: done.

File: test/e2e/apps/network_partition.go [hunk 3]
```diff
@@ -207,11 +207,11 @@ var _ = SIGDescribe("Network Partition [Disruptive] [Slow]", func() {
 				ginkgo.By(fmt.Sprintf("Block traffic from node %s to the master", node.Name))
 				host, err := e2enode.GetExternalIP(&node)
 				framework.ExpectNoError(err)
-				masterAddresses := framework.GetAllMasterAddresses(c)
+				controlPlaneAddresses := framework. GetAllControlPlaneAddresses(c)
```
Comment (oomichi) [inline test/e2e/apps/network_partition.go:210]: need to remove unnecessary space after `framework.`

File: test/e2e/apps/network_partition.go [hunk 4]
```diff
@@ -595,11 +595,11 @@ var _ = SIGDescribe("Network Partition [Disruptive] [Slow]", func() {
...
```
Comment (oomichi) [inline test/e2e/apps/network_partition.go:598]: ditto

File: test/e2e/framework/network/utils.go [hunk 5]
```diff
@@ -947,25 +947,25 @@ func TestUnderTemporaryNetworkFailure(c clientset.Interface, ns string, node *v1
 	if err != nil {
 		framework.Failf("Error getting node external ip : %v", err)
 	}
-	masterAddresses := framework.GetAllMasterAddresses(c)
+	controlPlaneAddresses := framework. GetAllControlPlaneAddresses(c)
```
Comment (oomichi) [inline test/e2e/framework/network/utils.go:950]: ditto

File: test/e2e/network/firewall.go [hunk 6]
```diff
@@ -216,10 +216,10 @@ var _ = SIGDescribe("Firewall rule", func() {
 			framework.Failf("did not find any node addresses")
 		}
 
-		masterAddresses := framework.GetAllMasterAddresses(cs)
-		for _, masterAddress := range masterAddresses {
-			assertNotReachableHTTPTimeout(masterAddress, ports.InsecureKubeControllerManagerPort, firewallTestTCPTimeout)
-			assertNotReachableHTTPTimeout(masterAddress, kubeschedulerconfig.DefaultInsecureSchedulerPort, firewallTestTCPTimeout)
+		controlPlaneAddresses := framework. GetAllControlPlaneAddresses(cs)
```
Comment (oomichi) [inline test/e2e/network/firewall.go:219]: ditto

**Comment by spiffxp** [comment]
/hold
ref: https://github.com/kubernetes/kubernetes/issues/94901#issuecomment-702344182

**Comment by spiffxp** [comment]
If you want to rebase and address review comments then it's ok to /reopen this. Otherwise will identify this as a change someone else can take on with a new PR

**Pull request closed without merging**